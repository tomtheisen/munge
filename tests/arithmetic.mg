(
    /.*/ => @ 1#(
        /\(\d+\)/ => (
            /\D/ => ""
        )
        1( /(\d+) *\* *(\d+)/ => { $1 $2 * } )
        1( 
            /(\d+) *\+ *(\d+)/ => { $1 $2 + } 
            /(\d+) *\- *(\d+)/ => { $1 $2 - } 
        )
    )
)