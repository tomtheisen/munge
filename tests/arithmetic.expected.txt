70
7
9
12
7